test/passing/js_source.ml.ref

```diff
@@ -1231,7 +1231,8 @@ let rec devariantize : type t e. e ty_env -> (t, e) ty -> variant -> t =
        | TCarg (sel, t), Some a -> ops.sum_inj (sel, devariantize e t a)
        | TCnoarg sel, None -> ops.sum_inj (sel, Noarg)
        | _ -> raise VariantMismatch
-     with Not_found -> raise VariantMismatch)
+     with
+    | Not_found -> raise VariantMismatch)
```

Comment:
misaligned?

Reply:
I fixed this alignment but now it is aligned with the `match` instead of the `(` for some other cases when it would have been broken anyway, I prefer this new formatting but added a condition on the hvbox to have a smaller diff.

Reply:
The last commit has been reverted.